src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -22,4 +22,107 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-{% use '@PrestaShop/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig' %}
+{% use 'bootstrap_4_horizontal_layout.html.twig' %}
```

Comment:
@eternoendless on my phpstorm setup it expects full path for 'bootstrap_4_horizontal_layout.html.twig'. Better to modify phpstorm settings or add a full path here?

Reply:
Just a thought that as we already have it: https://github.com/PrestaShop/PrestaShop/blob/f05a93b4434388fbeceb149331524bc3918574e2/app/config/config.yml#L55-L61

so, maybe we can remove bootstrap_4_horizontal_layout import here?

Reply:
Actually leaving the path relative in prestashop_ui_kit will actually make it use the `bootstrap_4_horizontal_layout.html.twig` file from Symfony instead of the one from PrestaShop. I didn't find any way to make it explicit :\

We need to keep the default form_theme like that until all forms have been migrated to the new form theme. Once that's done, we can switch the default and remove the old files, as I said in the PR description.